### DISCUSSION TITLE
How to run Windows app with admin previledge.

### DISCUSSION THREAD

**Discussion by yutapatio** [The-Wineskin-Project/WineskinServer#237] [Q&A] [answered]
Hi, I'm quite new about Wine.

I have installed Wineskin on M1 mac, and installed Autopsy, which is a forensic software.

The installation is successful, but this Autopsy requires "administrator" privilage to run properly.

Is it possible to run this application with privilage?

 

Thank you.

**Comment by Gcenx** [comment, marked as answer]
As that’s an upstream issue there’s not really anything I can do

**Comment by yutapatio** [comment]
I see. So there is no way to do that, right? Does anyone tell me about this?

**Reply by Gcenx** [reply]
wine _always_ runs application/games as administrator, some applications expect to be running as a normal user and elevate to administer this isn’t something that wine has implemented.

**Reply by yutapatio** [reply]
Thank you. Well, Autopsy is a special application which makes a snapshot of the disk image, so it needs administrator priviledge to lock the drive. I understand that all applications run as administrator, so I try to ask a question at github Autopsy community.

**Reply by Gcenx** [reply]
The application might simply fail to function under wine but yes asking the upstream project would be best

**Reply by yutapatio** [reply]
Thank you very much!
